docs/integrations/app-development/opentelemetry/jira.md

```diff
@@ -0,0 +1,225 @@
+---
+id: jira-opentelemetry
+title: Jira - OpenTelemetry Collector
+sidebar_label: Jira - OTel Collector
+description: The Sumo Logic App for Jira provides insight into Jira user access, request activity, issues, security, sprint events, and user events.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/app-development/jira.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+The Sumo Logic App for Jira provides insight into Jira usage, request activity, issues, security, sprint events, and user events. 
+
+Jira logs are sent to Sumo Logic through OpenTelemetry [filelog receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver).
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Jira-OpenTelemetry/Jira-Schematics.png')} alt="Schematics" />
+
+## Fields creation in Sumo Logic for Jira
+
+Following are the tags which will be created as part of the Jira App install if not already present. 
+
+`sumo.datasource` - Has fixed value of **jira**.
+
+## Prerequisites
+
+This section provides instructions for configuring log collection for Jira running on a non-Kubernetes environment for the Sumo Logic App for Jira. Sumo Logic supports the collection of logs from Jira server in standalone environments.
+
+Follow the instructions to set up log collection.
+
+The Jira app uses the following log types:
+-   **Jira Access Logs**. HTTP server access log files, in `logs/access_logs`,  in your JIRA Installation Directory.
+-   **Jira Security Logs**.Security-related information, such as logins, logouts, session creation/destruction, and security denials from `atlassian-jira-security.log`.
+-   **Jira Catalina Logs**. The application catalina log file, `logs/catalina.out`, in your JIRA Installation Directory.
+-   **Jira Webhooks**. Webhook events of types:
+    -   Issue
+    -   User
+    -   Sprint
+
+## Configure hosted collector to receive Webhooks
+
+In this step, you create a host collector to receive webhooks from Jira, and set up an HTTP source on it.
+
+1.  Configure a [hosted collector](https://help.sumologic.com/docs/send-data/hosted-collectors/configure-hosted-collector/), or select an existing hosted collector for the HTTP source.
+2.  Configure an [HTTP source](https://help.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/) on the hosted collector.
+    -   For Source Category, specify Atlassian/Jira/Events
+    -   Add `sumo.datasource` as File to this source and assign the value **jira** for it.
+    <img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Jira-OpenTelemetry/Jira-Datasource.png')} alt="Datasource" />
```

Comment:
This image link is broken when I run it in a local build. Verify that it works in your system.